app/decorators/member_decorator.rb

```diff
@@ -0,0 +1,7 @@
+class MemberDecorator < Draper::Decorator
+  delegate_all
+
+  def show_email
+    User.find(object.user_id).email
```

Comment:
Are you sure that can't be happen?
You didn't add `dependant: :destroy` for association user has_many :members so when you destroy user (somehow) then you will get exception.